vpr/src/route/multi_queue_priority_queue.h

```diff
@@ -7,47 +7,35 @@
 
 #include "rr_graph_fwd.h"
 
-struct pq_node_t {
-    float cost = 0.;
-    float backward_path_cost = 0.;
-    float R_upstream = 0.;
-
-    RRNodeId index = RRNodeId::INVALID(); // TODO: to be optimized
-
-    uint32_t prev_edge_id;
-
-    pq_node_t() {}
-    pq_node_t(size_t dont_care) { (void)dont_care; } // for MQ compatibility
+struct node_t {
```

Comment:
I really like the use of this struct. But it doesnt appear to be used in this file. Could you move it either to the header file or implementation of the parallel connection router? Since I think this is more used to organize things within there.